docs/data/joy/getting-started/templates/email/components/Header.tsx

```diff
@@ -0,0 +1,271 @@
+          size="sm"
+          sx={{ alignSelf: 'center' }}
+        >
+          Team
+        </Button>
+        <Button
+          variant="plain"
+          color="neutral"
+          component="a"
+          href="/joy-ui/getting-started/templates/files/"
+          size="sm"
+          sx={{ alignSelf: 'center' }}
+        >
+          Files
+        </Button>
+      </Stack>
+      <Box sx={{ display: { xs: 'inline-flex', sm: 'none' } }}>
+        <IconButton variant="plain" color="neutral" onClick={() => setOpen(true)}>
+          <MenuRoundedIcon />
+        </IconButton>
+        <Drawer
+          sx={{ display: { xs: 'inline-flex', sm: 'none' } }}
+          open={open}
+          onClose={() => setOpen(false)}
+        >
+          <ModalClose />
+          <DialogTitle>Acme Co.</DialogTitle>
+          <Box sx={{ px: 1 }}>
+            <Navigation />
+          </Box>
+        </Drawer>
+      </Box>
+
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'row',
+          gap: 1.5,
+          alignItems: 'center',
+        }}
+      >
+        <Input
+          size="sm"
+          variant="outlined"
+          placeholder="Search anything…"
+          startDecorator={<SearchRoundedIcon color="primary" />}
+          endDecorator={
+            <IconButton
+              variant="outlined"
+              color="neutral"
+              sx={{ bgcolor: 'background.level1' }}
+            >
+              <Typography level="title-sm" textColor="text.icon">
+                ⌘ K
+              </Typography>
+            </IconButton>
+          }
+          sx={{
+            alignSelf: 'center',
+            display: {
+              xs: 'none',
+              sm: 'flex',
+            },
+          }}
+        />
+        <IconButton
+          size="sm"
+          variant="outlined"
+          color="neutral"
+          sx={{ display: { xs: 'inline-flex', sm: 'none' }, alignSelf: 'center' }}
+        >
+          <SearchRoundedIcon />
+        </IconButton>
+        <Tooltip title="Joy UI overview" variant="outlined">
+          <IconButton
+            size="sm"
+            variant="plain"
+            color="neutral"
+            component="a"
+            href="/blog/first-look-at-joy/"
+            sx={{ alignSelf: 'center' }}
+          >
+            <BookRoundedIcon />
+          </IconButton>
+        </Tooltip>
+        <ColorSchemeToggle />
+        <Dropdown>
+          <MenuButton
+            variant="plain"
+            size="sm"
+            sx={{ maxWidth: '32px', maxHeight: '32px', borderRadius: '9999999px' }}
+          >
+            <Avatar
+              src="https://i.pravatar.cc/40?img=2"
+              srcSet="https://i.pravatar.cc/80?img=2"
+              sx={{ maxWidth: '32px', maxHeight: '32px' }}
+            />
+          </MenuButton>
+          <Menu
+            placement="bottom-end"
+            size="sm"
+            sx={{
+              zIndex: '99999',
```

Comment:
Is this necessary?

Reply:
If I remove the `zIndex` the dropdown end up hidden behind the header 😬 

| With `zIndex` | Without `zIndex` |
|--------|--------|
| <img width="219" alt="Screenshot 2023-11-02 at 12 18 16" src="https://github.com/mui/material-ui/assets/37222944/485ebf78-c9b4-4f17-a784-c46971141b46"> | <img width="249" alt="Screenshot 2023-11-02 at 12 18 04" src="https://github.com/mui/material-ui/assets/37222944/82c60caf-d6bc-4870-b93d-656c576e386c"> |